src/main/resources/static/api.yaml

```diff
@@ -2455,11 +2511,25 @@ components:
           description: '4 digit number'
         scope:
           type: string
-          enum : [consentrequest.approve, consent.revoke]
+          enum: [consentrequest.approve, consent.revoke, profile.changepin]
           description: 'Scope the supplied token will be valid for'
       required:
         - pin
         - scope
+    ChangePinRequest:
+      type: object
+      properties:
+        requestId:
```

Comment:
Do we need requestId here? Thought we discussed as to we don't need one.

Reply:
we will be just sending it over..won't be using it any were..there will be one different card to validate this request-id to avoid replay attacks..

Reply:
But, can there be replay attacks for this api? 
As it will need to get the temp token before you perform this action.

Reply:
yeah..actually..I checked..there is a cache where we can put the used tokens..and then it checks if the token is used already..
I was unaware of this cache thing..

Reply:
We will be sending request-id here. As the way of handling replay attack is gonna change in future. So, that time we will need request-id.

Reply:
@gayatrijagtap From our discussion we have decided to not to add request-id as of now.

Reply:
removed request-id